Framework/include/QualityControl/Activity.h

```diff
@@ -31,6 +31,8 @@ namespace o2::quality_control::core
 /// \author   Barthelemy von Haller
 class Activity
 {
+  // TODO make it a struct ?
```

Comment:
no harm in that

Reply:
I am just wondering if there is an impact on the Serialization with ROOT